src/plugins/workspace/common/constants.ts

```diff
@@ -37,3 +40,42 @@ export const PRIORITY_FOR_WORKSPACE_ID_CONSUMER_WRAPPER = -3;
 export const PRIORITY_FOR_WORKSPACE_UI_SETTINGS_WRAPPER = -2;
 export const PRIORITY_FOR_WORKSPACE_CONFLICT_CONTROL_WRAPPER = -1;
 export const PRIORITY_FOR_PERMISSION_CONTROL_WRAPPER = 0;
+
+export const WORKSPACE_APP_CATEGORIES: Record<string, AppCategory> = Object.freeze({
+  // below categories are for workspace
+  getStarted: {
+    id: 'getStarted',
+    label: i18n.translate('core.ui.getStarted.label', {
```

Comment:
I think the i18n might complain about the namespaces as it will expect `workspace`. Can we verify it with:
```
node ./scripts/i18n_extract.js --output-dir ./translations
```

Reply:
There will be errors but ignore the unrelated ones.

Reply:
@kavilla Yes, that will be error to complain it need workspace namespace.
``` Error: Expected "core.ui.getStarted.label" id to have "workspace" namespace. See .i18nrc.json for the list of supported namespaces.```

Reply:
create a follow up issue https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6647